tests/test_config.py

```diff
@@ -0,0 +1,217 @@
+import argparse
+import os
+import unittest
+
+import autograder.api.config
+import autograder.util.dir
+import autograder.util.dirent
+
+THIS_DIR = os.path.abspath(os.path.dirname(os.path.realpath(__file__)))
+CONFIGS_DIR = os.path.join(THIS_DIR, "data", "configs")
+
+TESTS = {
```

Comment:
Why is this a module-level constant?
Can it not just live in the function where it is used?